pkgs/development/web/woff2/brotli-dependencies.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ecfbb83..da8ecf1 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -21,16 +21,6 @@ option(NOISY_LOGGING "Noisy logging" ON)
+ # Version information
+ set(WOFF2_VERSION 1.0.2)
+ 
+-# When building shared libraries it is important to set the correct rpath
+-# See https://cmake.org/Wiki/CMake_RPATH_handling#Always_full_RPATH
+-set(CMAKE_SKIP_BUILD_RPATH FALSE)
+-set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+-list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_LIBDIR}" isSystemDir)
+-if ("${isSystemDir}" STREQUAL "-1")
+-  set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR}")
+-endif()
+-
+ # Find Brotli dependencies
+ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+ find_package(BrotliDec)
+diff --git a/cmake/FindBrotliDec.cmake b/cmake/FindBrotliDec.cmake
+index abb06f4..a56d4b5 100644
+--- a/cmake/FindBrotliDec.cmake
++++ b/cmake/FindBrotliDec.cmake
+@@ -18,10 +18,10 @@ find_path(BROTLIDEC_INCLUDE_DIRS
+     HINTS ${PC_BROTLIDEC_INCLUDEDIR}
+ )
+ 
+-find_library(BROTLIDEC_LIBRARIES
+-    NAMES brotlidec
+-    HINTS ${PC_BROTLIDEC_LIBDIR}
+-)
++if(NOT BUILD_SHARED_LIBS)
++  set(_S "STATIC_")
++endif()
++set(BROTLIDEC_LIBRARIES ${PC_BROTLIDEC_${_S}LIBRARIES})
```

Comment:
Can we send this upstream? Or at least report the issue?

Reply:
I opened https://gitlab.kitware.com/cmake/cmake/issues/20136 instead, with that, a cleaner solution is possible. I'm just wary that google open source projects often don't reply to PRs. See
https://github.com/google/woff2/pull/112 for example, I hoped that this patch would fix our problem, but that did not work for me.

Reply:
Great!
https://github.com/google/woff2/pull/112 is the ticket.